src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -628,6 +628,35 @@
      * be variable length (indicated by the ELLIPSIS child node immediately
      * after the TYPE child).
      *
+     * <p>For example</p>
+     * <pre>
+     * public int add(int num1, int num2)
```

Comment:
add `{}` after the method declaration

Reply:
Do I have to write a complete function there? like { return num1 + num2;} as well?

Reply:
You can make the function return type as void and just add empty method body.